go/ct/common/hash.go

```diff
@@ -15,7 +15,7 @@ import (
 	"pgregory.net/rand"
 )
 
-func GetRandomHash(rnd *rand.Rand) tosca.Hash {
+func GetRandomHash(rnd *rand.Rand) tosca.Hash { // coverage-ignore nothing to test
```

Comment:
two things, why do we need this? 
second: I tried it locally, and it does not ignore my code, is it correct?

Reply:
this comment was the first try to exclude/ignore files/pkg/blocks. Also this file should not be considered for testing since it has only one function that calls to `rnd.Read`, so it would be like testing that function.

Reply:
I do not think this is a good argument for why this should not be tested. Please remove the `coverage-ignore` comment.

A test just running it to make sure it does not crash would be sufficient in this case.

Reply:
a test has been added for this in the follow-up PR #639